install_template/config.yaml

```diff
@@ -81,9 +78,6 @@ products:
       - name: Ubuntu 22.04
         arch: x86_64
         supported versions: [55]
-      - name: Ubuntu 20.04
```

Comment:
Why Ubuntu 20 is removed from Migration Toolkit?

Reply:
@nidhibhammar That was a mistake. Should be fixed now.